README.md

```diff
@@ -68,6 +68,18 @@ Value | Explanation
 `replace` | you can't have the same numbered bookmark in different files
 `allowDuplicates` | you can have the same numbered bookmark in different files, and if you jump repeatedly to the same number, it will look on other files
 
+* You can change the background color of a bookmark using color names `blue`, RGB `rgb(0, 255, 37)` or HEX `#00ff25` format.
+
+```json
+    "numberedBookmarks.fillColor"
```

Comment:
Rename the setting to `gutterIconFillColor` to be more clear that this _fillColor_ is related to the gutter icon. It also be similar to the other `gutterIconPath` from my other Bookmarks extension [link](
https://github.com/alefragnani/vscode-bookmarks/#available-settings)